GOVERNANCE-elections.md

```diff
@@ -0,0 +1,274 @@
+# Steering Committee Governance
+
+This is an example governance document for projects that use steering committee
+elections from the contributors at large.  See [GOVERNANCE.md](/GOVERNANCE.md)
+for more information.  Thanks to the Kubernetes Project for portions of the text
+below.
+
+This is a template document for CNCF projects that requires editing
+before it is ready to use. Read the markdown comments, `<!-- COMMENT -->`, for
+additional guidance. The raw markdown uses `TODO` to identify areas that
+require customization.  Replace PROJECTNAME with the name of your project.
+
+<!-- template begins here-->
+
+The PROJECTNAME Steering Committee is the governing body of the PROJECTNAME project,
+providing decision-making and oversight pertaining to the project bylaws,
+sub-organizations, and financial planning. The Steering Committee also defines
+the project values and structure.
+
+This governance  is an open, living document, and will continue to
+evolve as the community and project change.
+
+- [Charter](#charter)
+- [Delegated authority](#delegated-authority)
+- [Committee Meetings](#committee-meetings)
+- [Committee Members](#committee-members)
+- [Decision process](#decision-process)
+- [Getting in touch](#getting-in-touch)
+- [Elections](#election-procedure)
+  - [Timeline](#timeline)
+  - [Election Officer(s)](#election-officers)
+  - [Eligibility to Vote](#eligibility-to-vote)
+  - [Candidate Eligibility](#candidate-eligibility)
+  - [Voting Procedure](#voting procedure)
```

Comment:
Fix broken link

```suggestion
  - [Voting Procedure](#voting-procedure)
```

Reply:
The link is still broken?

Reply:
it shouldn't be.  hmmmm.... feh, change to that file didn't get picked up.  force pushing again.

Reply:
OK, it's actually fixed now.